concepticondata/conceptlists/Zhou-2020-350.tsv

```diff
@@ -0,0 +1,351 @@
+Zhou-2020-350-200	200	to answer	答應	1275	ANSWER
+Zhou-2020-350-201	201	to arrive	到	1387	ARRIVE
+Zhou-2020-350-202	202	to ask, to tell	問、告訴	1309	ASK (INQUIRE)
+Zhou-2020-350-203	203	to be able to	會	972	BE ABLE
+Zhou-2020-350-204	204	to be willing to	肯		
+Zhou-2020-350-205	205	to become	變成	1990	BECOME
+Zhou-2020-350-206	206	to beg	乞討	3534	BEG
+Zhou-2020-350-207	207	to belong to a Chinese zodiac	屬（豬）		
+Zhou-2020-350-208	208	to bite	咬、吃	1403	BITE
+Zhou-2020-350-209	209	to blink	眨	1655	BLINK
+Zhou-2020-350-210	210	to block	堵塞	2762	BLOCK (THE WAY)
+Zhou-2020-350-211	211	to blow	吹	175	BLOW (OF WIND)
+Zhou-2020-350-212	212	to boil	煮	2492	BOIL (SOMETHING)
+Zhou-2020-350-213	213	to break (from the middle)	斷	3020	BREAK (BREAKING)
+Zhou-2020-350-214	214	to broke in the middle	破、斷		
+Zhou-2020-350-215	215	to burn	燃燒	2102	BURN
+Zhou-2020-350-216	216	to burn	燒	2102	BURN
+Zhou-2020-350-217	217	to carry	搬	700	CARRY
+Zhou-2020-350-218	218	to carry on the back	背 v.	465	CARRY ON BACK
+Zhou-2020-350-219	219	to chisel	鑿	1979	CHISEL
+Zhou-2020-350-220	220	to chop	剁	155	CHOP
+Zhou-2020-350-221	221	to comb	梳	2070	COMB (VERB)
+Zhou-2020-350-222	222	to compel	逼		
+Zhou-2020-350-223	223	to connect	連接	2862	CONNECT
+Zhou-2020-350-224	224	to contain	裝	697	CONTAIN
+Zhou-2020-350-225	225	to cough	咳嗽	879	COUGH
+Zhou-2020-350-226	226	to cover up	蓋上	1719	BURY
+Zhou-2020-350-227	227	to crawl	爬、上	96	CRAWL
+Zhou-2020-350-228	228	to crow	啼	2074	CROW (VERB)
+Zhou-2020-350-229	229	to cry	哭	1839	CRY
+Zhou-2020-350-230	230	to cut with a saw	鋸		
+Zhou-2020-350-231	231	to cut with knife	割		
+Zhou-2020-350-232	232	to die	死	1494	DIE
+Zhou-2020-350-233	233	to do	做	813	DO
+Zhou-2020-350-234	234	to dream	夢	1920	DREAM (SOMETHING)
+Zhou-2020-350-235	235	to drip	撒、滴	808	DRIP (FALL IN GLOBULES)
+Zhou-2020-350-236	236	to drive away	趕	30	DISPEL
+Zhou-2020-350-237	237	to eat	吃	1336	EAT
+Zhou-2020-350-238	238	to escape	逃	1615	ESCAPE
+Zhou-2020-350-239	239	to exchange	交換	2509	EXCHANGE
+Zhou-2020-350-240	240	to exit	出	1107	GO OUT
+Zhou-2020-350-241	241	to expose under sun	曬		
+Zhou-2020-350-242	242	to fear	怕	1419	FEAR (BE AFRAID)
+Zhou-2020-350-243	243	to feed	餵	739	FEED
+Zhou-2020-350-244	244	to feed	飼養	739	FEED
+Zhou-2020-350-245	245	to flip over	翻過來		
+Zhou-2020-350-246	246	to float	浮	1574	FLOAT
+Zhou-2020-350-247	247	to fly	飛	1441	FLY (MOVE THROUGH AIR)
+Zhou-2020-350-248	248	to forget	忘記	1523	FORGET
+Zhou-2020-350-249	249	to fry	油炸	2162	STIR FRY
+Zhou-2020-350-250	250	to get	得到	694	GET
+Zhou-2020-350-251	251	to give, to feed	給、送、餵	1447	GIVE
+Zhou-2020-350-252	252	to go pass	過 v.		
+Zhou-2020-350-253	253	to grip	握	1448	HOLD
+Zhou-2020-350-254	254	to grow	生長	479	GROW
+Zhou-2020-350-255	255	to grow fruit	結（果子）		
+Zhou-2020-350-256	256	to grow up	長大	552	GROW UP
+Zhou-2020-350-257	257	to hang	掛	2884	HANG
+Zhou-2020-350-258	258	to hear	聽見	1408	HEAR
+Zhou-2020-350-259	259	to hold a crutch	拄		
+Zhou-2020-350-260	260	to infect	傳染		
+Zhou-2020-350-261	261	to itch	癢	3856	ITCH OR ITCHY OR ITCHING
```

Comment:
[BeijingDaxue-1964-905-771](https://concepticon.clld.org/values/BeijingDaxue-1964-905-771) has the same gloss and is mapped to ITCHY. Should this be remapped?

Reply:
@LinguList Should I remap this?

Reply:
Yes, itchy is a good candidate.